operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakDeploymentTest.java

```diff
@@ -477,7 +513,7 @@ public void testUpgradeRecreatesPods() {
                         assertEquals(origImage, sts.getSpec().getTemplate().getSpec().getContainers().get(0).getImage());
 
                         var currentKc = k8sclient.resources(Keycloak.class)
-                                        .inNamespace(namespace).withName(kc.getMetadata().getName()).get();
+                                .inNamespace(namespace).withName(kc.getMetadata().getName()).get();
```

Comment:
I believe this change is out-of-scope of this PR. Please revert if possible.

Reply:
Done! Thanks for your review and comment.